installer/ClientInstaller.cs

```diff
@@ -79,33 +80,29 @@ public class ClientInstaller : MumbleInstall {
 		if (arch == "x64") {
 			// 64 bit
 			this.Platform = WixSharp.Platform.x64;
-			binaries = new string[] {
-				"celt0.0.7.0.dll",
-				"opus.dll",
-				"rnnoise.dll",
-				"speex.dll",
-				"mumble.exe",
-				"mumble_app.dll",
-				"mumble_ol.dll",
-				"mumble_ol_helper.exe",
-				"mumble_ol_helper_x64.exe",
-				"mumble_ol_x64.dll",
-				"mumble-g15-helper.exe"
-			};
+			binaries.add("celt0.0.7.0.dll");
+			binaries.add("opus.dll");
+			binaries.add("rnnoise.dll");
+			binaries.add("speex.dll");
+			binaries.add("mumble.exe");
+			binaries.add("mumble_app.dll");
+			binaries.add("mumble_ol.dll");
+			binaries.add("mumble_ol_helper.exe");
+			binaries.add("mumble_ol_helper_x64.exe");
+			binaries.add("mumble_ol_x64.dll");
+			binaries.add("mumble-g15-helper.exe");
```

Comment:
A C# List<T> can also be initialized with an initialization body `{ el1, el2 }`

Reply:
The array was chosen to be more like C++ syntax so it would be easier to work with. Feel free to add it as a list, but I would recommend doing it as @Kissaki suggested so we don't have to prefix it with `binaries.add()` for each new entry.

Reply:
Ah I did not know that. Will do :+1: